MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiDxeServicesTableLib.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+  Google Test mocks for DxeServicesTableLib
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef MOCK_UEFI_DXE_SERVICES_TABLE_LIB_H_

```

Comment:
Naming convention is to use the library name you are mocking, ie
`DxeServicesTableLib` -> `MOCK_DXE_SERVICES_TABLE_LIB_H_` with no `UEFI`

Reply:
Change library name from MockUefiDxeServicesTableLib to MockDxeServicesTableLib